packages/logs/src/domain/logger.ts

```diff
@@ -52,7 +52,7 @@ export class Logger {
           })
           break
         case HandlerType.console:
-          console.log(`${status}: ${message}`)
+          console.log(`${status}: ${message}`, combine(this.contextManager.get(), messageContext))
```

Comment:
We could think about including the [other attributes that should end up in the log eventually](https://github.com/DataDog/browser-sdk/blob/master/packages/logs/src/boot/logs.entry.ts#L113).
However, this seems to be the simplest thing to do for now and having the other attributes could also unnecessarily bloat the console context.
@BenoitZugmeyer, @hdelaby any opinion on that?

Reply:
I think user-defined context is enough here.

* `date` is the current timestamp in millisecond: not very user friendly.
* `view.url` is (in general) the URL shown in the browser URL bar.
* `view.referrer` is (in general) an information known by the developer since it's their navigation, and this information coulb be useful in very particular situations.

Reply:
I am not sure to understand what's at stake here. Can you elaborate a bit? 🙏

Reply:
With this PR current implementation, when logs are send to the console, the displayed context:

- contains the log context
- contains the logger context
- does not contain global context
- does not contain page, session, date info
- does not contain RUM internal context

are we happy with this behavior or is there some part that we would want to be present in the displayed context?

Reply:
as seen together ok for now, and we'll add APIs to retrieve current global context.